RELEASES.md

```diff
@@ -1,3 +1,132 @@
+- [`ops::RangeFull`, and `ops::Range` now implement Default.][73197]
+- [`panic::Location` now implements `Copy`, `Clone`, `Eq`, `Hash`, `Ord`,
+  `PartialEq`, and `PartialOrd`.][73583]
+
+Stabilized APIs
+---------------
+- [`Ident::new_raw`]
+- [`Range::is_empty`]
+- [`RangeInclusive::is_empty`]
+- [`Result::as_deref`]
+- [`Result::as_deref_mut`]
+- [`Vec::leak`]
+- [`pointer::offset_from`]
+- [`f32::TAU`]
+- [`f64::TAU`]
+
+The following previously stable APIs have now been made const.
+
+- [`TypeId::of`]
+- [The `new` method for all `NonZero` integers.][73858]
+- [The `checked_add`,`checked_sub`,`checked_mul`,`checked_neg`, `checked_shl`,
+  `checked_shr`, `saturating_add`, `saturating_sub`, and `saturating_mul`
+  methods for all integers.][73858]
+- [The `checked_abs`, `saturating_abs`, `saturating_neg`, and `signum`  for all
+  signed integers.][73858]
+- [The `is_ascii_alphabetic`, `is_ascii_uppercase`, `is_ascii_lowercase`,
+  `is_ascii_alphanumeric`, `is_ascii_digit`, `is_ascii_hexdigit`,
+  `is_ascii_punctuation`, `is_ascii_graphic`, `is_ascii_whitespace`, and
+  `is_ascii_control` methods for `char` and `u8`.][73858]
+
+Cargo
+-----
+- [`build-dependencies` are now built with opt-level 0 by default.][cargo/8500]
+  You can override this by setting the following in your `Cargo.toml`.
+  ```toml
+  [profile.release.build-override]
+  opt-level = 3
+  ```
+- [`cargo-help` will now display man pages for commands rather just the
+  `--help` text.][cargo/8456]
+- [`cargo-metadata` now emits a `test` field indicating if a target has
+  tests enabled.][cargo/8478]
+- [`workspace.default-members` now respects `workspace.exclude`.][cargo/8485]
+- [`cargo-publish` will now use an alternative registry by default if it's the
+  only registry specified in `package.publish`.][cargo/8571]
+
+Misc
+----
+- [Added the `rustc-docs` component.][75560] This allows you to install
+  and read the documentation for the compiler internal APIs. (Currently only
+  available for `x86_64-unknown-linux-gnu`.)
+- [Added a help button beside Rustdoc's searchbar that explains rustdoc's
+  type based search.][75366]
+
+Compatibility Notes
+-------------------
+- [The `missing_fragment_specifier` lint has been converted to a hard error.][75516]
+  This lint had been set to `deny` by default since 2017.
+- [Bumped the minimum supported Emscripten version to 1.39.20.][75716]
+- [Bumped the minimum supported LLVM version to 10.0.0.][75908]
+- [Fixed a regression parsing `{} && false` in tail expressions.][74650]
+- [Added changes to how proc-macros are expanded in `macro_rules!` that should
+  help to preserve more span information.][73084] These changes may cause
+  compiliation errors if your macro was unhygenic or didn't correctly handle
+  some delimiter information.
+- [Moved support for the CloudABI target to tier 3.][75568]
+
+Internal Only
+--------
+- [Improved default stages of the compiler to use for each of the commands
+  in `x.py`.][73964]
```

Comment:
That PR mentions that separate work would be needed for documentation -- did that happen? It would be helpful to include documentation links for this change if possible.

Reply:
https://blog.rust-lang.org/inside-rust/2020/08/30/changes-to-x-py-defaults.html

Reply:
```suggestion
  in `x.py`.][73964] This major change to the compiler was accompanied by a [blog post](https://blog.rust-lang.org/inside-rust/2020/08/30/changes-to-x-py-defaults.html).
```

Reply:
Also I don't think 'improve default stages' is a good description, it did a lot more than switch the default stage from 2 to 1. I've just been saying 'improve x.py default' but maybe for release notes you could say 'improved default settings for bootstrapping'?